setup.py

```diff
@@ -67,7 +67,7 @@
 # General Requirements
 INSTALL_REQUIRES += [
     "sympy>=1.6, < 1.7",
-    "django >= 1.8, < 1.12",
+    "django >= 3.0",
```

Comment:
Please let's put a cap on this like `<4.0`. (If it were me I'd've made the cap 3.2. Otherwise there is likely automatic breakage. when 3.2 or 4.0 come out. 

Thanks again.